src/app/+dashboard/components/personal-achievements/personal-achievements.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <div class="header">
+    <p>PERSONAL ACHIEVEMENTS</p>
+    <button>Request</button>
+  </div>
+
+  <div *ngFor="let achievment of achievments" class="achievment">
+    <div class="ach">
+      <img alt="" src="{{ achievment.img }}">
+
+      <div class="achievment-text">
+        <p class="achievment-title">{{ achievment.title }}</p>
+        <p class="achievment-time">{{ achievment.time }} mins ago</p>
```

Comment:
Achivement time will be passed as a date type. So it would be nice if you provide a pipe here, where you will pass achievement added date and will get appropriate string (values that are required here are described in doc i shared with you)

Reply:
fixed